devenv/docker/blocks/jwt_proxy/docker-compose.yaml

```diff
@@ -33,22 +33,26 @@
     image: docker.io/bitnami/oauth2-proxy:7.3.0
     container_name: oauthproxy
     command: [
-    "--cookie-secret=yI-CWT5s4sBR2Zd0DDJJlTYc0aQ3jwGH15jYA18ZAQA=",
-    "--upstream=http://localhost:3000",
-    "--provider=keycloak",
-    "--client-id=grafana-oauth",
-    "--client-secret=d17b9ea9-bcb1-43d2-b132-d339e55872a8",
-    "--login-url=http://127.0.0.1:8087/auth/realms/grafana/protocol/openid-connect/auth",
-    "--redeem-url=http://127.0.0.1:8087/auth/realms/grafana/protocol/openid-connect/token",
-    "--profile-url=http://127.0.0.1:8087/auth/realms/grafana/protocol/openid-connect/userinfo",
-    "--validate-url=http://127.0.0.1:8087/auth/realms/grafana/protocol/openid-connect/userinfo",
-    "--cookie-secure=false",
-    "--http-address=0.0.0.0:8088",
-    "--redirect-url=http://127.0.0.1:8088/oauth2/callback",
-    "--pass-access-token=true",
-    "--email-domain=*",
+      "--cookie-secret=yI-CWT5s4sBR2Zd0DDJJlTYc0aQ3jwGH15jYA18ZAQA=",
+      "--upstream=http://env.grafana.local:3000",
+      "--provider=keycloak",
+      "--client-id=grafana-oauth",
+      "--client-secret=d17b9ea9-bcb1-43d2-b132-d339e55872a8",
+      "--login-url=http://env.grafana.local:8087/auth/realms/grafana/protocol/openid-connect/auth",
+      "--redeem-url=http://env.grafana.local:8087/auth/realms/grafana/protocol/openid-connect/token",
+      "--profile-url=http://env.grafana.local:8087/auth/realms/grafana/protocol/openid-connect/userinfo",
+      "--validate-url=http://env.grafana.local:8087/auth/realms/grafana/protocol/openid-connect/userinfo",
+      "--cookie-secure=false",
+      "--http-address=0.0.0.0:8088",
+      "--redirect-url=http://env.grafana.local:8088/oauth2/callback",
+      "--pass-access-token=true",
+      "--email-domain=*",
     ]
-    network_mode: "host"
+    # network_mode: "host"
     depends_on:
       - oauthkeycloak
+    extra_hosts:
+      - "env.grafana.local:host-gateway"
```

Comment:
i have not seen this. does this work for all os? 🤔

Reply:
tested on linux working here :+1: 

@mgyongyosi tested mac and windows

Reply:
It isn't documented extensively, but 'host-gateway' should resolve to the host's address on every platform.